### ISSUE TITLE
implement getter to fetch ballot creation info from Ethereum and IPFS

### ISSUE THREAD

**Issue by virgile-dev** [DemocracyEarth/dapp#50]
Ballot information that we need to query:
- [x] title
~author name~
- [x] date of creation
- [x] transaction status

The transaction status is only needed when the user creates the ballot. I'm thinking to construct the page we don't really have to get title, author name, date of creation if we just cache them and construct the page with the info. That way the only getter we'd need would be the transaction status. What do you think @LucasIsasmendi @aecc ?

**Comment by virgile-dev** [comment]
The ballot page when just created:
<img width="784" alt="capture d ecran 2018-04-22 a 12 48 03" src="https://user-images.githubusercontent.com/11473995/39094318-e1124a32-462d-11e8-8d8b-e78bdd3902d2.png">

**Comment by AlexJupiter** [comment]
If the proposed changes are implemented in https://github.com/DemocracyEarth/contracts/issues/51 I think this issue is redundant right?

**Comment by aecc** [comment]
Still the we need to implement fetching title, author, date from the blockchain. I'll be working on this soon

**Comment by LucasIsasmendi** [comment]
title, author and date are not coming from IPFS?

**Comment by aecc** [comment]
yes, I meant from IPFS, through the blockchain first. As I see only author name is missing. The rest, title and date seem implemented

**Comment by AlexJupiter** [comment]
@aecc @LucasIsasmendi there seem to be some duplicates of the points here in https://github.com/DemocracyEarth/dapp/issues/49, should this issue be closed and we just refer to #49 ?

Also, is this in "in progress", and if so, which one of you is working on this?

**Comment by LucasIsasmendi** [comment]
There is a pending "author name" implementation, the trick here is if we should rely on IPFS or ETH for that. But yes, we can close this because we changed the view model of this ballot detail page

**Comment by LucasIsasmendi** [comment]
You are right @aecc , reopening this to complete pending task
